common.blocks/i-bem/i-bem.spec.js

```diff
@@ -187,14 +193,14 @@ describe('i-bem', function() {
                 block.hasMod('mod1').should.be.true;
             });
 
-            it('in short form should return true for empty mod\'s value', function() {
+            it('in short form should return false for empty mod\'s value', function() {
```

Comment:
I'm not sure about why there was "true" if `shoud.be.false` is checked inside the test?